src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -63,7 +63,7 @@ public static IEnumerable<object[]> SslStream_StreamToStream_Authentication_Succ
             using (X509Certificate2 clientCert = Configuration.Certificates.GetClientCertificate())
             {
                 yield return new object[] { new X509Certificate2(serverCert), new X509Certificate2(clientCert) };
-                yield return new object[] { new X509Certificate(serverCert.Export(X509ContentType.Pfx)), new X509Certificate(clientCert.Export(X509ContentType.Pfx)) };
+                yield return new object[] { new X509Certificate(serverCert.Export(X509ContentType.Pfx), (string)null, X509KeyStorageFlags.DefaultKeySet), new X509Certificate(clientCert.Export(X509ContentType.Pfx), (string)null, X509KeyStorageFlags.DefaultKeySet) };
```

Comment:
If you pass ReadOnlySpan(byte or char).Default (or ""u8) you'll hit one of the span ctors, which default the key storage flags.  So this line can be shorter:

```suggestion
                yield return new object[] { new X509Certificate(serverCert.Export(X509ContentType.Pfx), ""u8), new X509Certificate(clientCert.Export(X509ContentType.Pfx), ""u8) };
```

Reply:
Ah, the others aren't defaulted, because they're overloaded.

```suggestion
                yield return new object[] { new X509Certificate(serverCert.Export(X509ContentType.Pfx), ""), new X509Certificate(clientCert.Export(X509ContentType.Pfx), "") };
```

also works.  (or `(string?)null`, if you prefer)